Formula/fstar.rb

```diff
@@ -37,16 +38,20 @@ def install
 
     system "opam", "config", "exec", "--", "make", "-C", "src", "boot-ocaml"
 
-    bin.install "bin/fstar.exe"
+    bin.install "bin/fstar.exe" => "_fstar.exe"
```

Comment:
That keeps it in `bin/`, doesn't it? 
I end up with two files in `#{bin}`: the fstar executable in `#{bin}/_fstar.exe` and the script that the formula writes to `#{bin}/fstar.exe`.